docs/source/dev/sphinx.rst

```diff
@@ -0,0 +1,65 @@
+.. _development-sphinx:
+
+Sphinx
+======
+
+This page explains how to contribute to this documentation.
```

Comment:
OPTIONAL:
double use of `this` - perhaps replace second `this` by `our` or `PIConGPU's`